internal/metrics/collector.go

```diff
@@ -68,7 +68,7 @@ var (
 	databaseSizeDesc = prometheus.NewDesc(
 		metricPrefix+"database_size_bytes",
 		"Size of database in bytes as reported from engine.",
-		nil, nil)
+		[]string{"version", "type"}, nil)
```

Comment:
I think it would match the style of the other metrics more if this information would be surfaced in an "info metric" similar to the other meta-information.

Reply:
done, hope that's what you meant